Formula/asciiquarium.rb

```diff
@@ -0,0 +1,55 @@
+class Asciiquarium < Formula
+  desc "Aquarium animation in ASCII art"
+  homepage "http://robobunny.com/projects/asciiquarium/html/"
+  url "http://www.robobunny.com/projects/asciiquarium/asciiquarium_1.1.tar.gz"
+  sha256 "1b08c6613525e75e87546f4e8984ab3b33f1e922080268c749f1777d56c9d361"
+
+  resource "Curses" do
+    url "https://cpan.metacpan.org/authors/id/G/GI/GIRAFFED/Curses-1.34.tar.gz"
+    mirror "http://search.cpan.org/CPAN/authors/id/G/GI/GIRAFFED/Curses-1.34.tar.gz"
+    sha256 "808e44d5946be265af5ff0b90f3d0802108e7d1b39b0fe68a4a446fe284d322b"
+  end
+
+  resource "Term::Animation" do
+    url "http://robobunny.com/projects/animation/Term-Animation.tar.gz"
```

Comment:
Is this a static URL? i.e. always points to the latest `Term-Animation` release?